content/blog/2021-12-15-Feuchtigkeit-Teutoburger-Wald.md

```diff
@@ -0,0 +1,60 @@
+---
+type: blog # So lassen
+title: "Bodenfeuchtigkeit sichtbar machen: Innovationen für den Teutoburger Wald"
+author: Meike Wocken
+twitter_handle: "@CountessOfCount"
+excerpt: "In Bielefeld wird die Feuchtigkeit des heimischen Waldbodens mit Sensoren gemessen. Ziel ist es, dafür eine Citizen Science-Community aufzubauen und die Daten für alle als Open Data und aufbereitet in einer Online-Karte zur Verfügung zu stellen."
+topic: community, project
+images:
+ - imgname: teuto_wald2.jpg
+ - imgname: teuto_team.jpg # Dieses Bild sollte im Verzeichnis static/blog existieren
+ - imgname: teuto_ranger_sensor.jpg
```

Comment:
Ok sorry ich bin ein bisschen benebelt vom Booster, der erste Eintrag hier wird als Bild vom Blogpost in der Übersicht angezeigt, dem Pfad sollte aber `feuchtigkeit-teutoburger-wald/` vorrangestellt sein

Reply:
Kein Problem, hätte ich auch dran denken können :-)